src/tests/ftest/util/MpioUtils.py

```diff
@@ -88,3 +88,50 @@ def run_romio(self, basepath, hostlist, romio_test_repo):
 
         except (IOError, OSError, paramiko.SSHException, socket.error) as e:
             raise MpioFailed("<ROMIO Test FAILED> \nException occurred: {}".format(str(e)))
+
+    def run_ior(self, basepath, pool_uuid, svcl, np, hostfile, ior_flags, iteration,
+                transfer_size, block_size, display_output=True):
+        """
+            Running IOR over mpich
+            basepath       --Daos basepath
+            pool_uuid      --Daos Pool UUID
+            svcl           --Daos Pool SVCL
+            np             --number of client processes
+            hostfile       --client file holding client hostname and slots
+            ior_flags      --all ior specific flags
+            iteration      --number of iterations for ior run
+            block_size     --contiguous bytes to write per task
+            transfer_size  --size of transfer in bytes
+            display_output --print IOR output on console.
+        """
+        try:
+            # environment variables only to be set on client node
+            env_variables = ["export CRT_ATTACH_INFO_PATH={}/install/tmp/".format(basepath),
+                             "export DAOS_POOL={}".format(pool_uuid), "export MPI_LIB=''",
+                             "export DAOS_SVCL={}".format(svcl), "export DAOS_SINGLETON_CLI=1",
+                             "export FI_PSM2_DISCONNECT=1"]
```

Comment:
This will expected to run only with PSM2? We should export this only if it's PSM2

Reply:
Yes we should be using psm2 and nvme for Performance runs.